app/model/release.js

```diff
@@ -0,0 +1,139 @@
+// Imports for packages
+import mongoose from 'mongoose';
+import Promise from 'bluebird';
+import semver from 'semver';
+
+// Local imports
+import app from '~/'
+import Jenkins from './jenkins';
+import IssueSchema from './issue';
+import { BuildSchema } from './build';
+
+// Mongoose schema for releases
+const ReleaseSchema = mongoose.Schema({
+  github: {                 // All Github communication data
+    id:       Number,       // Id for Github API
+    author:   String,       // Github user login
+    date:     Date,         // Github publish date
+    tag:      String,       // Github uncleaned tag (v4.2.3)
+  },
+  status: {                 // Build status
+    type:     String,
+    default: 'NEW RELEASE',
+    enum:   ['NEW RELEASE', 'BUILDING', 'REVIEWING', 'SUCCESS', 'FAILED'],
+  },
+  changelog: [String],      // List of changelog items for release
+  issue:      IssueSchema , // Current Github issue for release
+  problems: [{              // All release problems
+    type:     String,
+    enum: [
+      'changelogExist',     // Missing desktop file
```

Comment:
nevermind — I was conflating the app-level problems with the release-level problems!